helix-core/src/test/java/org/apache/helix/manager/zk/TestZkBaseDataAccessor.java

```diff
@@ -159,6 +159,43 @@ record = new ZNRecord("msg_1");
     System.out.println("END " + testName + " at " + new Date(System.currentTimeMillis()));
   }
 
+  @Test
+  public void testAsyncSetChildrenWithTTL() {
```

Comment:
Would like to see more description in detailed steps for reader to understand what does the test do.

Reply:
done, could you check again. thanks!

Reply:
It would be great to add more comments in the test, like steps 1, 2, 3 etc.

Reply:
ack, will add.

Reply:
added